### ISSUE TITLE
Unexpected token error after dependencies update

### ISSUE THREAD

**Issue by juffalow** [facebook/relay#3649]
I upgraded one of my projects to the latest `relay` version maybe two weeks ago and everything worked. I tried the same now and I get `Unexpected token` error.

In every `__generated__/*graphql.ts` file there is:

```ts
import { FragmentRefs as  } from "relay-runtime";
```

And correct is:

```ts
import { FragmentRefs } from "relay-runtime";
```

I upgraded dependencies in my previous project and I have the same error also there, so it is probably related to some dependency of `relay`.

**Comment by maraisr** [comment]
Can you raise this over at https://github.com/relay-tools/relay-compiler-language-typescript the js compiler has been deprecated. 

In case I'm wrong; can you post your relay, relay compiler, graphql and runtime versions. Thanks!

**Comment by juffalow** [comment]
package.json

```json
{
  "dependencies": {
    "babel-plugin-relay": "^12.0.0",
    "react-relay": "^12.0.0",
    "relay-runtime": "^12.0.0"
  },
  "devDependencies": {
    "@types/react-relay": "^11.0.2",
    "graphql": "^15.0.0",
    "relay-compiler": "^12.0.0",
    "relay-compiler-language-typescript": "^14.3.0",
    "typescript": "^4.0.2"
  },
  "scripts": {
    "relay": "relay-compiler --schema schema.graphql --src src --extensions tsx --language typescript"
  },
}
```

**Comment by juffalow** [comment]
You can find the whole project here: https://github.com/juffalow/react-relay-example

This one is working, but I upgraded dependencies locally just now and it is not working anymore.

**Comment by maraisr** [comment]
I'll check it out in the morning, and advise 👌🏻

**Comment by juffalow** [comment]
Thank you! :)

**Comment by juffalow** [comment]
Ok, it seems that the problem is on `relay-compiler-language-typescript` side https://github.com/relay-tools/relay-compiler-language-typescript/issues/513

**Comment by 8byr0** [comment]
For anyone coming here from google, this issue is fixed in relay-compiler-language-typescript@15.0.0. To fix, update your package.json or run

```bash
yarn add relay-compiler-language-typescript@^15.0.0
```
